enclave_wrangler/objects_api.py

```diff
@@ -344,6 +358,33 @@ def get_new_cset_and_member_objects(
     return {'cset_containers': cset_containers, 'cset_versions': cset_versions_with_concepts}
 
 
+# todo: call concept_set_members_enclave_to_db()
+def csets_and_members_to_db(con: Connection, csets_and_members: Dict[str, List] = None):
+    """Update database with csets and members."""
+    # For each obj: Insert
+    # todo: convert row for each table needed
+    #  - I will *maybe* want to refactor this so that uses modified fetch_object_and_add_to_db() which only takes
+    #  object_type_name, then uses a dict to figure out which tables to update and how
+
+    # TODO: I know the object types. so can I use the single functions to insert into DB?
+    #  - is the db update decoupled from the fetching? If not, do so
+    # table_obj: Dict = convert_row(object_type_name, table, obj)
+    # insert_from_dict(con, table, table_obj)
+
+    # def code_sets_enclave_to_db(con: Connection, object_id: int) -> Dict:
+    #     return fetch_object_and_add_to_db(con, 'code_sets', 'OMOPConceptSet', object_id)
+
+    # TODO: Utilize OBJECT_TYPE_TABLE_MAP?
+    OBJECT_TYPE_TABLE_MAP
+    print()
+
+
+def csets_and_members_enclave_to_db(con: Connection, csets_and_members: Dict[str, List] = None, since: Union[datetime, str] = None):
```

Comment:
WIP: This function will fetch any new updates for any of the following objects, and update their corresponding tables: `OMOPConceptSetContainer`, `OMOPConceptSet`, `OmopConceptSetVersionItem`, `OMOPConcept`